libcodechecker/analyze/analysis_manager.py

```diff
@@ -364,15 +371,41 @@ def check(check_data):
                         LOG.debug(str(ex))
                         other_files = set()
 
+                    mentioned_files_dependent_files = set()
```

Comment:
I would rather pick a name like `ctu_dependencies`. I think it might be more clear than mentioned files.

Reply:
There is an other path, where we get the files from a clang-tidy run.